### DISCUSSION TITLE
Is it possible to run Ant Media Server in the Termux app with Ubuntu?

### DISCUSSION THREAD

**Discussion by jmax07** [ant-media/Ant-Media-Server#5996] [Q&A]
I need help to install Anti Media Server on Termux with Ubuntu. I made a few attempts, but without success.
I can install it, but it doesn't work. Help me please!

**Comment by muratugureminoglu** [comment]
Duplicate question:

Explained [here](https://github.com/orgs/ant-media/discussions/6002#discussioncomment-8033033)
